client/components/Pipeline/Kanban.tsx

```diff
@@ -50,38 +51,25 @@ interface KanbanState {
   isLoading: boolean /* True if we are still grabbing data from api */;
   columns: Record<string, any>;
   columnOrder: string[];
-  partnerNames: Set<string>;
+  partners: Set<User>;
 }
 
 export default class Kanban extends PureComponent<KanbanProps, KanbanState> {
+  constructor(props) {
+    super(props);
+    this.loadCompanies = this.loadCompanies.bind(this);
+  }
+
   state = {
     isLoading: true,
     columns: {},
     columnOrder: [],
-    partnerNames: new Set([]),
+    partners: new Set([]),
   };
 
   async componentDidMount() {
-    try {
-      /* Get all companies that are not in archived state */
-      const res = (await client.service('api/companies').find({
-        query: {
-          status: {
-            $nin: archivedStates,
-          },
-          $eager: 'pointPartners',
-        },
-      })) as Paginated<Company>;
-      const ret = transformData(res.data);
-      this.setState({
-        columnOrder: ret.columnOrder,
-        columns: ret.columns,
-        partnerNames: ret.partnerNames,
-        isLoading: false,
-      });
-    } catch (error) {
-      console.error(error);
-    }
+    // should add 'default' as enum value to Team instead of giving as key directly?
+    this.loadCompanies('default', 'ALL');
```

Comment:
// should add 'default' as enum value to Team instead of giving as key directly?

Reply:
maybe later right - would also allow easier reset of team dropdown, would have to restrict this in the company/partnerTeamDropdown to prevent a company from being assigned to all teams

Reply:
Yea it should be an enum. Up to you whether to update or not

Reply:
Going to leave for now